src/heuristics.c

```diff
@@ -63,6 +63,10 @@ static bool is_state(char *filename)
 		return false;
 	}
 
+	if ((strlen(filename) == 15) && (strncmp(filename, "/usr/src/kernel", 15) == 0)) {
```

Comment:
This will *only* return false for the directory, but none of the files under that directory since you are only doing the strncmp for filenames that are exactly 15 characters long.

Reply:
OK,

So, Do  I need to just compare the first part of the path?

I will send an update.